azext_iot/iothub/_help.py

```diff
@@ -645,6 +645,146 @@ def load_iothub_help():
               --container {container_name} --encoding json --identity {user_identity_resource_id}
     """
 
+    helps[
+        "iot hub message-endpoint update"
```

Comment:
Added this as a way to organize params. I do not want the same mess as with the original routing-endpoint create.